kids-chat-app/.env.example

```
@@ -0,0 +1,24 @@
+# OpenAI Configuration
+OPENAI_API_KEY=your_openai_api_key_here
+OPENAI_API_URL=https://api.openai.com/v1
+
+# Database Configuration
+DATABASE_URL=postgresql://username:password@localhost:5432/kids_chat_db
+DATABASE_USER=your_db_username
+DATABASE_PASSWORD=your_db_password
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DATABASE_PASSWORD key should go before the DATABASE_URL key

(UnorderedKey)